solr/core/src/test/org/apache/solr/search/TestMinHashQParser.java

```diff
@@ -417,8 +417,7 @@ public void testBandsWrap() throws SyntaxError {
     QParser qparser =
         h.getCore()
             .getQueryPlugin("minhash")
-            .createParser(
-                "1, 2, 3, 4, 5, 6, 7, 8, 9, 10", SolrParams.toSolrParams(par), null, null);
+            .createParser("1, 2, 3, 4, 5, 6, 7, 8, 9, 10", par.toSolrParams(), null, null);
```

Comment:
"par"?   Sort of a nitpick, but maybe a more descriptive variable name?  "params'?